.env.example

```
@@ -0,0 +1,22 @@
+# .env.example - 환경 변수 설정 템플릿
+# 이 파일을 .env 로 복사한 후, 각 변수에 맞는 값을 입력하세요.
+
+# 1. OpenAI Settings
+# GPT-4o 모델 호출 및 임베딩 생성을 위한 인증 키
+OPENAI_API_KEY="YOUR_OPENAI_API_KEY"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)